compiler/rustc_middle/src/mir/pretty.rs

```diff
@@ -449,6 +449,7 @@ impl<'tcx> Visitor<'tcx> for ExtraComments<'tcx> {
             }
 
             let fmt_val = |val: &ConstValue<'tcx>| match val {
+                ConstValue::ZST => format!("ZST"),
```

Comment:
nit: the value isn't really a zero sized *type* :grin: idk what else I would want to print here though 

probably `<ZST>` which is what we've done for zero sized scalars before?

Reply:
Yeah, it's a ZSV, a zero-sized value. But we have called the value a ZST for a while now... (there's some red diff in this PR removing a `<ZST>` elsewhere; I think this will have to cause some test failure somewhere)

Reply:
If they're not too long, `ConstValue::ZeroSized` or `ConstValue::UnitLike` both seem fine to me.